src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -48,7 +48,8 @@ const InfoBox: UISchemaElementComponent<{
         variant="infobox"
         title={title}
         // eslint-disable-next-line react/jsx-props-no-spreading
-        {...(dataSe && { 'data-se': dataSe })}
+        {...({ 'data-se': dataSe })}
+        className={`infobox-${messageClass.toLowerCase()}`}
```

Comment:
I've been wondering if there's any way for us to differentiate between different alerts besides using class. Any ideas?

Reply:
The message type is structured like this:

```
{
   message: string;
   i18n: {
      key: string;
      params?: string[];
   };
   class: string;
```

Where class can be any of these: `ERROR`, `WARNING`, `INFO`

There are times however, where `INFO` is set when it actually needs to be displayed as an error (we determine that based on the i18n key). So not really sure how we can really determine what should be an error vs warning vs info message (if that was the question) without more information from the server side.

Reply:
I was referring to the ability for testing-library to differentiate between different `alert` boxes. Right now, there isn't much to differentiate the various alerts unless you inspect their class or CSS attributes.

Reply:
Ohhhhh, I'm sorry about that, so thinking and looking through the options available to `getByRole`, we could possibly use a hidden description element and using the aria-describedby property. This way, when we try to grab an error message by role, we can do so like this: 

```
getByRole('alert', { description: 'ERROR' });
```

Assuming the HTML looks like this 

```
<div role="alert" aria-describedby="error-desec">
   Some Error message
</div>
<span id="error-desec" aria-hidden="true">ERROR</span>
```

However, the only issue with this is that a screen reader will read out that `ERROR` description.